ui/pages/confirmations/hooks/alerts/useShieldCoverageAlert.ts

```diff
@@ -143,7 +143,7 @@ export function useShieldCoverageAlert(): Alert[] {
   const showAlert = isEnableShieldCoverageChecks;
 
   return useMemo<Alert[]>((): Alert[] => {
-    if (!showAlert) {
+    if (!showAlert || !status) {
```

Comment:
can you move status to the show alert condition above?

showAlert = isEnableShieldCoverageChecks && status